proto/grp.proto

```diff
@@ -28,6 +28,14 @@ message Group {
   string url_identifier = 5;
 }
 
+message JoinGroupRequest {
```

Comment:
We've been lax about it (my fault! sorry) but all Request/Response pairs in here need to have:

context.RequestContext request_context = 1;
in the request and 
context.ResponseContext response_context = 1;
in the response.

Could you do that here or in a followup?

Reply:
Sure, let me do that in a follow up